Missionframework/modules/06_virtual/README.md

```diff
@@ -0,0 +1,16 @@
+# KP Liberation Module Description
+
+## Zeus Module
+This module provides...
+
+### Dependencies
+* Init
+* Core
+
+### Functions
+* KPLIB_fnc_garrison_changeOwner
+
+  *Changes the owner of a sector garrison.*
```

Comment:
Wrong module
But as discussed, listing the functions again isn't needed. The functions can be looked up in the functions.hpp and all global variables in the preInit.
Things we could/should have in the module readme are running "endless" scripts explained and all eventhandlers which are fired in this module at which situations with which provided args. It would make it easier to hook into the events of a module with an extension.

Reply:
Oh yeah, forgot to update this after copy pasting the file 😒